lib/tarball.js

```diff
@@ -0,0 +1,58 @@
+const { relative } = require('path')
+
+const pkgContents = require('@npmcli/installed-package-contents')
+const pacote = require('pacote')
+const isPackageBin = require('pacote/lib/util/is-package-bin.js')
```

Comment:
yeesh... this is scary.

Maybe `pacote` should just expose its tar c options?

Reply:
If it did, then we get rid of the deep import, and the tarOpts function becomes:

```js
const tarOpts = manifest => ({
  ...pacote.tarCOpts(manifest),
  cwd: manifest._resolved,
})
```

Reply:
hmmm maybe `pacote.tarball.cOpts` ?

I'll follow up with a PR there 😊